src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './Components/Calculator';
+import './app.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="application">
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Great job here 💯 But according to the requirements, you should use class based components (All components should be class-based). so Kindly, the `App.js` component should be `class-based`. 👍